native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -106,6 +105,10 @@ const styles = StyleSheet.create({
     flexGrow: 1,
     paddingLeft: spacingSm,
     color: colorGray700,
+    paddingTop: ((spacingXl - (2 * borderSizeSm)) - lineHeightSm) / 2,
```

Comment:
What does this come out as? I'm wondering if there's a way we can simplify this line, and the ones beneath.